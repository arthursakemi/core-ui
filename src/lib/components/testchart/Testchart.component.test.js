import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/testchart/__snapshots__"
});
