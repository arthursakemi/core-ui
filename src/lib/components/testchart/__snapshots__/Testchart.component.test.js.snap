// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Testchart Default 1`] = `
<div>
  <h3>
    Vega-Lite linechart demo
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-testchart"
  >
    <div
      id="vis"
    />
  </div>
</div>
`;
